lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -14,15 +15,12 @@
 }
 
 .FilePickerApp__heading {
-  grid-column: 1 / 3;
-  margin-bottom: 1em;
+  grid-column: 1 / span 2;
```

Comment:
This is entirely dev preference here; it was perfectly valid before. I find `grid-column: 1 / span 2;` more intuitive than `grid-column: 1 / 3;` because I always have to look up that the `3` here is a grid-line. My brain wants to read it as `one-third` or, at least, `one of three`, which isn't correct. Thoughts?